.github/workflows/ci.yml

```diff
@@ -27,7 +33,7 @@ jobs:
         uses: conda-incubator/setup-miniconda@v3
         with:
           environment-file: ci/env.yaml
-          python-version: ${{ matrix.PY }}
+          python-version: ${{ matrix.python-version }}
```

Comment:
This was previously causing all tests to run on `conda-incubator/setup-miniconda`'s default Python (i.e. 3.12)

Reply:
oops